### PR TITLE
Add remote cmds and change how inetutils is packaged 

### PR DESCRIPTION
**Pull request by asdfugil**
### All Submissions

* [x] Have you made sure there aren't any other open [Pull Requests](https://github.com/ProcursusTeam/Procursus/pulls) for the same update/change?
* [ ] This Pull Request doesn't contain any package additions; it's a small change (e.g README change)

### Package Additions/Updates

* [x] Have you confirmed this builds & works as intended on an iOS device (if applicable)?
* [ ] Have you confirmed this builds & works as intended on a macOS device (if applicable)?

- Added remote-cmds
- Inetutils is split (debianization)
- Every Inetutils tool is prefixed with `inetutils-` (still sort of debianization, but Debian only prefix some)
- `hostname` is now shell-cmds hostname
 - Inetutils hostname is at `inetutils-hostname` (debianization)
- `ping` is now network-cmds ping
 - Likewise, Inetutils ping is still available at `inetutils-ping`
- launchd.plist written from scratch for every daemon added
- main Makefile: fix compiling when specificing multiple targets on command line

inetutils-ftpd and inetutils-telnetd works

Closes #999

### CODE REVIEW

**Comment by asdfugil** [comment]
telnet and ftp
AAA7D9B0-3C56-4E0D-88C3-3BE346319AB5

**Comment by Torrekie** [comment]
shall use program prefix 'g' rather than separate inetutils (inetutils-ftp -> gftp)

**Comment by CRKatri** [comment]
> shall use program prefix 'g' rather than separate inetutils (inetutils-ftp -> gftp)

No, debian uses the `inetutils-` prefix, so we will too.
https://packages.debian.org/sid/amd64/inetutils-ftp/filelist

**Comment by CRKatri** [comment]
I really appreciate you doing this. I haven't gotten to test it yet, but I do have one request. Don't start any of the daemons, they are all for daemons that are known to be insecure if not correctly configured. Still install them, but add the disabled key to them and don't start them in the maintainer scripts.

**Comment by asdfugil** [comment]
I can't see how `talkd` could be insecure.

**Comment by CRKatri** [comment]
Either way, don't enable it automatically and add the Disabled key.

**Comment by asdfugil** [comment]
When a package is installed: nothing happens other than its files being put in place

When `remove`d:
Daemons are unloaded

When `purge`d:
Daemons are disabled

File: build_info/inetutils-ftpd.prerm [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#\!/bin/sh
```
Comment (CRKatri) [inline build_info/inetutils-ftpd.prerm:1]:
```suggestion
#!/bin/sh
```
This is in the other `.prerm` scripts too.

**Comment by asdfugil** [comment]
For a given daemon package `food` with its `/usr/libexec/food` removed but its LaunchDaemons file is still retained as residual-config, there is a risk:
`panic(cpu 1 caller 0xfffffff00dc832b4): userspace panic: File Radar to: [ launchd | missing executable ]. Set boot-arg launchd_missing_exec_no_panic=1 to work around`

**Comment by asdfugil** [comment]
`build_tools/compile.sh` works

Updated inetutils to 2.2
Updated network-cmds to 624.100.3
Enable more stuff in inetutils since there is no name conflict now

**Pull request closed without merging**